src/rp2040/hardware_structs/include/hardware/structs/iobank0.h

```diff
@@ -45,12 +45,117 @@ typedef struct {
 
 typedef struct {
     _REG_(IO_BANK0_PROC0_INTE0_OFFSET) // IO_BANK0_PROC0_INTE0
+    // (Description copied from IO_BANK0_PROC0_INTE0 applies similarly all array members)
+    //
+    // Interrupt Enable for proc0
+    // 0x80000000 [31]    : GPIO7_EDGE_HIGH (0)
```

Comment:
For `inte[0]` this is `GPIO7_EDGE_HIGH`, for `inte[1]` this is `GPIO15_EDGE_HIGH`, for `inte[2]` it's `GPIO23_EDGE_HIGH` and for `inte[3]` it's `Reserved.`.
I wonder if it's worth adding a special-case to the svd2struct comment-generator-code for these `IO_BANK0` array-regs and instead just say something like `// Please refer to the RP2040 datasheet for full details` ? Which is a bit of a cop-out but I can't think of anything better off the top of my head. :confused:

Reply:
yeah you can just follow the _REG_ link to see the real registers